bom/build.gradle.kts

```diff
@@ -13,4 +13,4 @@ otelBom.addFallback("opentelemetry-exporter-jaeger-proto", "1.17.0")
 otelBom.addFallback("opentelemetry-extension-annotations", "1.18.0")
 otelBom.addFallback("opentelemetry-sdk-extension-resources", "1.19.0")
 otelBom.addFallback("opentelemetry-sdk-extension-aws", "1.19.0")
-otelBom.addFallback("opentelemetry-extension-aws", "1.20.0")
+otelBom.addFallback("opentelemetry-extension-aws", "1.20.1")
```

Comment:
is this right?

Reply:
It is the latest release in maven central

Reply:
oh sorry! I misread as `1.21.0` 🧐